net-proxy/mitmproxy/mitmproxy-5.2.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DISTUTILS_USE_SETUPTOOLS=rdepend
+PYTHON_COMPAT=( python3_{7,8} )
```

Comment:
Some deps are in the way of 3.9, right?

Reply:
Upstream also does not support Python 3.9. There is some work in the master branch and in some open pull requests.

Reply:
https://github.com/mitmproxy/mitmproxy/pull/4174

Reply:
Thanks, no worries. Re what you asked on IRC (should you add patches), the answer is yes, if they're not too invasive, but only if they're upstreamed ideally. Let's just remember to come back to it!